frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -831,6 +827,20 @@
             vm.loaderTitle = '';
             vm.newContainer = angular.element('.new-team-card');
 
+            // show loader
+            vm.startLoader = function(msg) {
+                vm.isLoader = true;
+                vm.loaderTitle = msg;
+                vm.newContainer.addClass('low-screen');
+            };
+
+            // stop loader
+            vm.stopLoader = function() {
+                vm.isLoader = false;
+                vm.loaderTitle = '';
+                vm.newContainer.removeClass('low-screen');
+            };
+
```

Comment:
For the right-side loader i.e. for _**Create Team**_ I've changed the `ng-show` variable from `challenge.isExistLoader`(which is already set for pagination _**"left-side loader"**_) to `challenge.isLoader` so for right side loader I've defined two functions(**startLoader & stopLoader**) for showing the loader.